src/shapes/Shape.js

```diff
@@ -0,0 +1,147 @@
+import { svgns } from '../canvas/Canvas';
+
+export class Shape {
+    /* NOTE: CURRENTLY DOESN'T SUPPORT MULTIPLE DOM ELEMENTS PER SHAPE */
+    constructor(canvas, name, attributes={}, svgTag='path',append=false) {
```

Comment:
do you set things like, `class=area` here? If not, why not?

Reply:
not yet, hopefully next pr

Reply:
right now it offers an easy-to-use api to do just that, so not too worried about it